src/tests/system/tests/test_sssctl.py

```diff
@@ -191,12 +191,14 @@ def test_sssctl__reset_cached_timestamps(client: Client, ldap: LDAP):
 @pytest.mark.importance("high")
 @pytest.mark.tools
 @pytest.mark.topology(KnownTopology.Client)
-def test_sssctl__check_typo_option_name(client: Client):
+@pytest.mark.parametrize("stopped_sssd", [True, False])
```

Comment:
Not a big deal but IMO it would be better to avoid parameterize here unless really needed as this effectively doubles the test runs + cleanup between tests. In this case can we just do the assertions, stop sssd then do the assertions again. I don't insist however, what do you think?

Reply:
I agree with you, I remove the parametrization